src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -19,32 +21,32 @@ interface Props {
 
 /**
  * Task component
+ * 
+ * @param props component properties
  */
 const TaskTable = ({project, loggedInpersonId, filter}: Props) => {
   const { tasksApi, timeEntriesApi } = useLambdasApi();
   const [tasks, setTasks] = useState<Tasks[]>([]);
   const [timeEntries, setTimeEntries] = useState<number[]>([]);
-  const [open, setOpen] = useState<boolean>(false);
-  const [loading, setLoading] = useState<boolean>(false);
+  const [open, setOpen] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const columns = sprintViewTasksColumns({tasks, timeEntries});
+  const setError = useSetAtom(errorAtom);
 
   /**
    * Gather tasks and time entries when project is available and update changes appeared
    */
   useEffect(() => {
-    const fetchData = async () => {
-      setLoading(true);
-      if (project && open) {
-        await getTasksAndTimeEntries();
-        setLoading(false);
-      }
-    };
-    fetchData();
-  }, [project, loggedInpersonId, filter, open]);
+    if (project && open) {
+    getTasksAndTimeEntries();
+    }
+    }, [project, loggedInpersonId, filter, open]);
 
   /**
    * Get tasks and total time entries  
    */
   const getTasksAndTimeEntries = async () => {
+    setLoading(true);
```

Comment:
Do you think we could also have a  check in here to see if the tasks and time entries data has already been loaded e.g. if the user opens and closes the project accordion it re fetches the data, but maybe we don't need to?